### PR TITLE
LPS-185953 - [CP 2.0] Automated Test

### PR DESCRIPTION
**Pull request by joelrocha68**
TIcket: [LPS-185953](https://liferay.atlassian.net/browse/LPS-185953)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro [hunk 1]
```diff
@@ -1,5 +1,11 @@
 definition {
 
+	macro assertCardTeamMebersActivation {
+		AssertElementPresent(
```
Comment (styller1) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2Utils.macro:4]: The member name is missing an M, the correct one should be "TeamMembers"

**Review by styller1 — CHANGES_REQUESTED** [review]

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:stop

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:osb-site-initializer-customer-portal:nocompile

**Pull request closed without merging**